Jenkinsfile

```diff
@@ -181,9 +187,9 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
               // itself. Otherwise there'd be no other way to retrieve the
               // artifacts. But note we only keep one build at a time.
               utils.shwrap("""
-              rm -rf /srv/devel/${devel_prefix}/build
-              mkdir -p /srv/devel/${devel_prefix}/build
-              cp -a \$(realpath builds/latest) /srv/devel/${devel_prefix}/build
+              rm -rf ${devel_builddir}
+              mkdir -p ${devel_builddir}
+              cp -aT builds ${devel_builddir}
```

Comment:
you made this change so we would include `builds.json` now?

Reply:
Yup, exactly!